lib/hammer_cli_katello/activation_key.rb

```diff
@@ -146,14 +146,19 @@ class SubscriptionsCommand < HammerCLIKatello::ListCommand
         field :id, _("ID")
         field :product_name, _("Name")
         field :format_consumed, _("Attached")
-        field :amount, _("Quantity")
+        field :quantity_attached, _("Quantity")
         field :start_date, _("Start Date"), Fields::Date
         field :end_date, _("End Date"), Fields::Date
         field :support_level, _("Support")
         field :contract_number, _("Contract")
         field :account_number, _("Account")
       end
 
+      def extend_data(data)
+        data["format_consumed"] = data["quantity"] == -1 ? _("#{data['consumed']} out of Unlimited") : data["quantity"]
```

Comment:
This isn't quite right. `data["quantity"]` is the total, so this column should be more like this:

```rb
data["format_consumed"] = _("#{data['consumed']} out of #{data['quantity'] == -1 ? 'Unlimited' : data['quantity']}")
```